src/InGAP.hs

```diff
@@ -0,0 +1,74 @@
+{-# LANGUAGE FlexibleInstances, FlexibleContexts #-}
+
+module InGAP where
```

Comment:
May be "ToGAP"? Like "ToString", "ToList", other conversions to something.

Reply:
Ok, I've fix it